src/index.js

```diff
@@ -0,0 +1,39 @@
+import home from './JS/homePage';
+import menu from './JS/menu';
+import head from './JS/header';
+import foot from './JS/footer';
+import contact from './JS/contact';
+
+const { div } = home();
+const {
+  headerContainer,
+  homeLink, menuLink,
+  contactLink,
+} = head();
+const content = document.querySelector('#content');
+content.appendChild(headerContainer);
+content.appendChild(div);
+
+function menuPage() {
+  content.innerHTML = '';
+  content.appendChild(headerContainer);
+  content.appendChild(menu());
+  content.appendChild(foot());
+}
+
+function homePage() {
+  content.innerHTML = '';
+  content.appendChild(headerContainer);
+  content.appendChild(div);
+}
+
+function contactPage() {
+  content.innerHTML = '';
+  content.appendChild(headerContainer);
+  content.appendChild(contact());
+  content.appendChild(foot());
+}
```

Comment:
Use arrow functions instead of the `function` keyword. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/))